src/tabs/tab-header-group.component.ts

```diff
@@ -211,6 +231,12 @@ export class TabHeaderGroup implements AfterContentInit, OnDestroy {
 		this.tabHeaderQuery.toArray()[this.currentSelectedIndex].selectTab();
 	}
 
+	ngOnChanges() {
+		if (this.tabHeaderQuery) {
+			this.tabHeaderQuery.toArray().forEach(tabHeader => tabHeader.cacheActive = this.cacheActive);
+		}
```

Comment:
probably want to check if `cacheActive` is actually in the changeset, otherwise this could end up running way more frequently than necessary